buildbuddy.yaml

```diff
@@ -107,4 +107,3 @@ plugins:
   - path: cli/plugins/open-invocation
   - path: cli/plugins/notify
   - path: cli/example_plugins/go-highlight
-  - path: cli/example_plugins/ping-remote
```

Comment:
@siggisim can I remove this plugin? Github runners don't support ping, so this plugin was preventing the build from using remote execution.

https://github.com/actions/runner-images/issues/1519

Reply:
Cool with me!